### PR TITLE
Define minimum OTP version and erl command output in data file

### PR DESCRIPTION
**Pull request by eksperimental**
Unify the minimum required OTP version and the output of the `erl`.
As you can see it was outdated in some places.

### CODE REVIEW

**Comment by eksperimental** [comment]
If this gets merged, I will be updating the instructions in the RELEASE.md in the Elixir project to streamline it.

**Comment by josevalim** [comment]
@eksperimental I like this! :heart:

Could we add min_elixir_guide_version: and min_mix_guide_version: as well? This way everything is centralized in one place and I know what I have to update when I want to change any of them. :)

**Comment by eksperimental** [comment]
sure we can. you mean min_elixir_guide_otp_verision?
I will add it tomorrow as I need to run now.

**Comment by josevalim** [comment]
> sure we can. you mean min_elixir_guide_otp_verision?

I mean a minimum elixir version to be used in the Elixir guide and the minimum elixir version to be used in the Mix guide.

File: getting-started/introduction.markdown [hunk 1]
```diff
@@ -3,6 +3,7 @@ layout: getting-started
 title: Introduction
 redirect_from: /getting_started/1.html
 ---
+{% assign stable = site.data.elixir-versions[site.data.elixir-versions.stable] %}
```
Comment (josevalim) [inline getting-started/introduction.markdown]: We can't use stable here because that's a branch, hence the minimim_elixir_version for the guide. :)

**Comment by josevalim** [comment]
Closing this as something similar has been pushed while I had forgotten about this.  Sorry @eksperimental and thanks!

**Pull request closed without merging**